### PR TITLE
Fix up recommendation page

### PR DESCRIPTION
**Pull request by ravindk89**
Staged views are in slack.

### CODE REVIEW

File: source/operations/checklists/hardware.rst [hunk 1]
```diff
@@ -46,23 +49,38 @@ While MinIO can run on commodity or "budget" hardware, we strongly recommend usi
...
        MinIO recommends a minimum of 128GB of memory per node for best performance.
 
    * - :octicon:`circle`
-     - Minimum of four nodes dedicated to object storage.
+     - .. cond:: k8s
+
+         MinIO requires a *minimum* of 4 worker nodes per MinIO Tenant
+
+         MinIO strongly recommends four distinct worker nodes dedicated to servicing the MinIO Tenant.
+         Colocating multiple high-performance softwares on the same nodes can result in resource contention and reduced overall performance for all softwares.
```
Comment (feorlen) [inline source/operations/checklists/hardware.rst:65]: `softwares` sounds odd to me, maybe `applications` or `services`?

Comment (ravindk89) [reply source/operations/checklists/hardware.rst:65]: services!

File: source/operations/checklists/hardware.rst [hunk 2]
```diff
@@ -138,10 +136,7 @@ The following examples of network throughput constraints assume spinning disks w
 
 - 1GbE network link can support up to 125MB/s, or one spinning disk
 - 10GbE network can support approximately 1.25GB/s, potentially supporting 10-12 spinning disk
```
Comment (feorlen) [inline source/operations/checklists/hardware.rst:138]: typo in original: disk -> disks

**Review by feorlen — APPROVED** [review]
One tiny typo in original, otherwise 👍🏻

**Pull request merged**